src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -3230,6 +3231,9 @@ private void ToCSharp(InternalRegressionTree tree, TextWriter writer, int node,
             }
         }
 
+        /// <summary>
+        /// Copy the weights of all training features to <paramref name="weights"/>.
+        /// </summary>
```

Comment:
></ [](start = 12, length = 2)

Missing pararm for weights

Reply:
Thanks. I will add
```csharp
        /// <param name="weights">a <see cref="VBuffer{T}"/> where feature weights would be assigned to.
        /// The i-th element in <paramref name="weights"/> stores the weight of the i-th feature.</param>
```

---
In reply to: [261459910](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261459910) [](ancestors = 261459910)